onnxscript/rewriter/ort_fusions/sdpa.py

```diff
@@ -132,20 +158,29 @@ def rewrite(
         self,
         op,
         query: ir.Value | None,
-        key_transposed: ir.Value | None,
+        key: ir.Value | None,
         value: ir.Value | None,
         mask: ir.Value | None,
+        key_format: str,
         **_,
     ):
-        sdpa_args = [query, key_transposed, value]
+        sdpa_args = [query, key, value]
         if mask is not None:
             sdpa_args.append(mask)
         # If the scale is None, SDPA will use the default scaling factor, which is 1/sqrt(head_size).
-        return op.SDPA(*sdpa_args, scale=self._scale, _domain="ai.onnxruntime.fusion")
+        return op.SDPA(
+            *sdpa_args,
+            scale=self._scale,
+            key_format=key_format,
+            _domain="ai.onnxruntime.fusion",
```

Comment:
I wonder if something like `ai.onnxruntime.__fusion__` or `ai.onnxruntime._fusion` will make it clear that this is internal use only

Reply:
Sure. The underscore makes sense. I wonder, though, about the "onnxruntime" part ... these rules can potentially be migrated to do fusion for opset23 standard onnx as well, at which point some of the fusions may not be ORT specific. Not sure if some other name would be better.

Reply:
Will do it separately to avoid need for another approval